src/app/app.component.html

```diff
@@ -3,6 +3,6 @@
 <div class="bcrumbs-wrap">
   <app-breadcrumbs [class.bcrumbs--hidden]="isHome"></app-breadcrumbs>
 </div>
-<router-outlet (activate)="onActivate($event)"></router-outlet>
+<router-outlet></router-outlet>
```

Comment:
Looking at this more closely, the issue was that `onActivate` doesn't expect any arguments. So we can restore the function and change this to
```suggestion
<router-outlet> (activate)="onActivate()"</router-outlet>
```
Although given that `onActivate` only scrolls to the corner, perhaps it's not worth having it? I'm only concerned in case leaving it out causes some hard-to-diagnose UI bug!

What do you think?

Reply:
I opted to remove the function in the end as the scroll didn't seem to be a vital UI feature. However it is possible that it was implemented for a reason, in which case we can leave it in there (either accepting the $event argument and catching that in the function, or removing the argument altogether as you have done).

I would probably opt for removal of the function unless there was an obvious UI bug, and if a bug is present we should document it.

Reply:
Okay, let's remove it.